### ISSUE TITLE
What % of human games are OZ and OX nets?

### ISSUE THREAD

**Issue by pondturtle** [pangafu/LeelaMasterWeight#19]
I am just asking out of interest because OX24 seems to be just playing zero-style opening most of the time.

Do the nets still follow naming scheme with G nets where "4" in "OX24" mean 40% oh human games? If so any chance we might get higher % of human games with reinforcement learning later? Thank you

**Comment by pangafu** [comment]
No... OZ-seiral is contain 20% - 40% human game, most for color stm.
OX-serial is oz self-training, and it turning into zero-style.
